packages/web/src/tests.js

```diff
@@ -78,10 +78,11 @@ async function showTest(ctx) {
     const test = await connection.one(getTestRunByID(id))
     const result = await connection.any(getTestResultByID(id))
     if (result.length === 0) { return ctx.render('tests/loading') }
-
+    const failedTestResults = result.filter(a => !a.test_output.passed)
+    const allTestsPassed = failedTestResults.length ? false : true
```

Comment:
```suggestion
    const allTestsPassed = result.some(t => !t.test_output.passed)
```

Reply:
this always returns true 🤔

Reply:
😕 it shouldn't :D

Reply:
Oh I have it the wrong way around, this would return true if 1 test failed. Add a bang `!` before `result.some`